OpenRA.Game/Traits/TraitsInterfaces.cs

```diff
@@ -477,10 +477,25 @@ public interface ISelection
 		void Clear();
 		bool RolloverContains(Actor a);
 		void SetRollover(IEnumerable<Actor> actors);
-		void DoControlGroup(World world, WorldRenderer worldRenderer, int group, Modifiers mods, int multiTapCount);
+	}
+
+	public interface IControlGroupsInfo : ITraitInfoInterface
+	{
+		string[] Groups { get; }
+	}
+
+	public interface IControlGroups
+	{
+		string[] Groups { get; }
```

Comment:
I don't understand why this is required. Can't you just `WorldActor.Trait<IControlGroups>().Info.Groups` and avoid the redundancy?

Reply:
This was suggested by pchote and it seems to me like the correct way to do it (otherwise we're assuming knowledge of the implementation of `ControlGroupsInfo`) :thinking: Do we even have an actual world actor during linting?